src/script/components/DeviceCard.tsx

```diff
@@ -0,0 +1,135 @@
+import NamedIcon from './NamedIcon';
+import LegalHoldDot from './LegalHoldDot';
+
+export interface DeviceCardProps {
+  click?: (device: ClientEntity) => void;
+  device: ClientEntity;
+  showIcon?: boolean;
+  showVerified?: boolean;
+}
+
+const DeviceCard: React.FC<DeviceCardProps> = ({
+  click,
+  device: wrappedDevice,
+  showVerified = false,
+  showIcon = false,
+}) => {
+  const clientEntity = wrappedDevice;
+  const {class: deviceClass = '?', id = '', label = '?', meta} = clientEntity;
+  const formattedId = id ? clientEntity.formatId() : [];
+  const name = clientEntity.getName();
+  const clickable = !!click;
+  const isVerified = meta.isVerified;
+  const showLegalHoldIcon = showIcon && deviceClass === ClientClassification.LEGAL_HOLD;
+  const showDesktopIcon = showIcon && deviceClass === ClientClassification.DESKTOP;
+  const showOtherIcon = showIcon && !showLegalHoldIcon && !showDesktopIcon;
+
+  const clickOnDevice = () => {
+    if (typeof click === 'function') {
+      click(clientEntity);
+    }
+  };
+
+  return (
+    <div
+      className={cx('device-card', {'device-card__no-hover': !clickable})}
+      onClick={clickOnDevice}
+      data-uie-uid={id}
+      data-uie-name="device-card"
+    >
+      {showLegalHoldIcon && (
+        <LegalHoldDot
+          className="device-card__icon device-card__legal_hold_icon"
+          dataUieName="status-legal-hold-device"
+        />
+      )}
+      {showDesktopIcon && (
+        <NamedIcon
+          width={16}
+          height={16}
+          name="desktop-icon"
+          className="device-card__icon"
+          data-uie-name="status-desktop-device"
+        />
+      )}
+      {showOtherIcon && (
+        <NamedIcon
+          width={16}
+          height={16}
+          name="devices-icon"
+          className="device-card__icon"
+          data-uie-name="status-mobile-device"
+        />
+      )}
+      <div className="device-card__info" data-uie-name="device-card-info" data-uie-value={label}>
+        <div className="label-xs">
+          <span className="device-card__model">{name}</span>
+        </div>
+        <div className="text-background label-xs">
+          <span>{t('preferencesDevicesId')}</span>
+          <span data-uie-name="device-id">
+            {formattedId.map(id => (
+              <span key={id} className="device-id-part">
+                {id}
+              </span>
+            ))}
+          </span>
+        </div>
+      </div>
+      {showVerified && isVerified() && (
+        <NamedIcon
+          width={14}
+          height={16}
+          className="verified-icon"
+          name="verified-icon"
+          data-uie-name="user-device-verified"
+        />
+      )}
+      {showVerified && !isVerified() && (
+        <NamedIcon
+          width={14}
+          height={16}
+          className="not-verified-icon"
+          name="not-verified-icon"
+          data-uie-name="user-device-not-verified"
+        />
+      )}
```

Comment:
Might be a good candidate for a component

Reply:
TBH to me it does not look so much work to be a separate component

Reply:
We have various places where we want to show the verified icon in different states. So something like
```
<VerifiedIcon verified={isVerfiied()} />
```
would be nicer than repeating both icons over and over.

Reply:
done in https://github.com/wireapp/wire-webapp/pull/10436/commits/63ef9cca389f94481696d5b80c15e4185441a8fb